packages/notebook-extension/src/index.ts

```diff
@@ -422,10 +426,10 @@ const tabIcon: JupyterFrontEndPlugin<void> = {
       ) as HTMLLinkElement;
       switch (status) {
         case 'busy':
-          link.href = busyIcon;
+          link.href = baseURL + busyIcon;
```

Comment:
Should this string concatenation be moved above where `notebookIcon` and `busyIcon` are defined?

Reply:
Also it could be combined with `URLExt.join()` (some example uses in this repo).

Reply:
Thanks for the Quick review. This will be more readable. have Updated the PR.